ephios/static/ephios/js/main.js

```diff
@@ -1,4 +1,10 @@
-
 $(function () {
-  $('[data-toggle="tooltip"]').tooltip()
-})
\ No newline at end of file
+    $('[data-toggle="tooltip"]').tooltip();
+
+    $("[data-formset]").formset({
+        animateForms: true,
+        reorderMode: 'dom',
+    }).on("formAdded", "div", function (event) {
+        $(event.target).find("select").first().djangoSelect2()
+    });
```

Comment:
So this fixes your JS and also avoids inline-JS.
Although you still should change the `select` selector to something more specific for this to be in `main.js`.

Reply:
but why? enabling select2 features on newly added select inputs seems like a good idea to me

Reply:
but I might not want *all* select input elements to be a select2 one. I'd rather have that stated explicitly, as it might lead to bugs or unwanted overhead later.

Reply:
The django-select2 widgets already come with a `django-select2` class, so I'd suggest to use `.django-select2` as the jquery selector.

Reply:
`$('.django-select2').djangoSelect2()` is also what the packaged JS does.